.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+nomad-spark PR Summary
+==============================
+
+Briefly describe how this Pull Request changes the project.
+
+Additional Details
+==============================
+
+If necessary, provide additional context or details about this Pull Request, including all changes not mentioned in the summary.
+
+Checklist
+==============================
+
+Testing
+-------------------------
+
+(Remove this checklist and replace it with "N/A - no code changes" if this PR does not modify source code)
+
+* [ ] I have manually verified that my code changes do the right thing.
```

Comment:
@scunniff 
can we add a item -
`Is the PR raised against branch ExpediaGroup:nomad-spark-2.4.3 `

Sometimes, the Reviewer may forget the rules, and may approve it.